terraform/main.tf

```diff
@@ -28,24 +28,25 @@ module "container_registry" {
 
 
 module "orchestration" {
-  source                       = "git::https://github.com/wri/gfw-terraform-modules.git//terraform/modules/fargate_autoscaling?ref=v0.3.0"
-  project                      = local.project
-  name_suffix                  = local.name_suffix
-  tags                         = local.tags
-  vpc_id                       = data.terraform_remote_state.core.outputs.vpc_id
-  private_subnet_ids           = data.terraform_remote_state.core.outputs.private_subnet_ids
-  public_subnet_ids            = data.terraform_remote_state.core.outputs.public_subnet_ids
-  container_name               = var.container_name
-  container_port               = var.container_port
-  desired_count                = var.desired_count
-  fargate_cpu                  = var.fargate_cpu
-  fargate_memory               = var.fargate_memory
-  auto_scaling_cooldown        = var.auto_scaling_cooldown
-  auto_scaling_max_capacity    = var.auto_scaling_max_capacity
-  auto_scaling_max_cpu_util    = var.auto_scaling_max_cpu_util
-  auto_scaling_min_capacity    = var.auto_scaling_min_capacity
-  security_group_ids           = [data.terraform_remote_state.core.outputs.postgresql_security_group_id]
-  task_role_policies           = []
+  source                    = "git::https://github.com/wri/gfw-terraform-modules.git//terraform/modules/fargate_autoscaling?ref=v0.3.0"
+  project                   = local.project
+  name_suffix               = local.name_suffix
+  tags                      = local.tags
+  vpc_id                    = data.terraform_remote_state.core.outputs.vpc_id
+  private_subnet_ids        = data.terraform_remote_state.core.outputs.private_subnet_ids
+  public_subnet_ids         = data.terraform_remote_state.core.outputs.public_subnet_ids
+  container_name            = var.container_name
+  container_port            = var.container_port
+  desired_count             = var.desired_count
+  fargate_cpu               = var.fargate_cpu
+  fargate_memory            = var.fargate_memory
+  auto_scaling_cooldown     = var.auto_scaling_cooldown
+  auto_scaling_max_capacity = var.auto_scaling_max_capacity
+  auto_scaling_max_cpu_util = var.auto_scaling_max_cpu_util
+  auto_scaling_min_capacity = var.auto_scaling_min_capacity
+  security_group_ids        = [data.terraform_remote_state.core.outputs.postgresql_security_group_id]
+  task_role_policies = [module.lambda_raster_tiler.lambda_invoke_policy_arn, module.storage.s3_write_tiles_arn,
+  module.lambda_raster_tiler.lambda_invoke_policy_arn]
```

Comment:
It looks to me like there are two instances of `module.lambda_raster_tiler.lambda_invoke_policy_arn` here. Should one be something else?